### ISSUE TITLE
Form validation: dropdowns not generated, only value verified

### ISSUE THREAD

**Issue by GiantCrocodile** [gi0baro/weppy#62]
If I create a model and define an int field with `validation =`

```
...
        "country_id": {
                        "presence": True,
                        'in': {'set': [0, 1], 'labels': ['Germany', 'Other']}
                      },
...
```

I would expect my dropdown menu to be displayed on registration. But it only shows a textfield where the input is only allowed to be 0 or 1. Any other value is denied. When I register with 0 or 1 and see at my profile I see that the value of country_id is "value".

Why is there no dropdown and why is country_id "value" and not 0 or 1 in my profile? If I look at my db, there is 1 or 0 stored, not "value" or nothing.

This is related to http://weppy.org/docs/0.5/validations#value-inclusion

**Comment by gi0baro** [comment]
@GiantCrocodile I will inspect this in the next hours.

**Comment by gi0baro** [comment]
@GiantCrocodile this is fixed on master and v0.5.3 release

**Comment by GiantCrocodile** [comment]
Thanks @gi0baro. I really appreciate your work and fast support!
